VERSION

```diff
@@ -1 +1 @@
-0.7.1
```

Comment:
@mouse-reeve I think the version was accidentally changed in this PR

Reply:
Oh my bad, I was changing it for testing and forgot to revert

Reply:
https://github.com/bookwyrm-social/bookwyrm/pull/3276